public/app/features/browse-dashboards/BrowseFolderAlertingPage.test.tsx

```diff
@@ -0,0 +1,193 @@
+import 'whatwg-fetch'; // fetch polyfill
+import { render as rtlRender, screen } from '@testing-library/react';
+import { rest } from 'msw';
+import { SetupServer, setupServer } from 'msw/node';
+import React from 'react';
+import { TestProvider } from 'test/helpers/TestProvider';
+
+import { contextSrv } from 'app/core/core';
+import { getRouteComponentProps } from 'app/core/navigation/__mocks__/routeProps';
+import { backendSrv } from 'app/core/services/backend_srv';
+import {
+  GrafanaAlertStateDecision,
+  PromAlertingRuleState,
+  PromRulesResponse,
+  PromRuleType,
+  RulerRulesConfigDTO,
+} from 'app/types/unified-alerting-dto';
+
+import BrowseFolderAlertingPage, { OwnProps } from './BrowseFolderAlertingPage';
+
+function render(...[ui, options]: Parameters<typeof rtlRender>) {
+  rtlRender(<TestProvider>{ui}</TestProvider>, options);
+}
+
+jest.mock('@grafana/runtime', () => ({
+  ...jest.requireActual('@grafana/runtime'),
+  getBackendSrv: () => backendSrv,
+  config: {
+    ...jest.requireActual('@grafana/runtime').config,
+    unifiedAlertingEnabled: true,
+  },
+}));
+
+const mockFolderName = 'myFolder';
+const mockFolderUid = '12345';
+const mockAlertRuleName = 'myAlertRule';
+
+const mockRulerRulesResponse: RulerRulesConfigDTO = {
```

Comment:
I would like to discuss this with the wider frontend group - is there a better home where these fixtures can live? Maybe we could put a TODO on this to find a better home for it?

Personally, I would like to extend this fixtures pattern we/I started in browse-dashboards to the rest of the codebase...

Reply:
have moved these into separate fixtures files, still in the `browse-dashboards` folder for now. should make it very easy to move them later (e.g. if we want to move the libraryElements fixtures to the libraryPanels folder)